format/index.md

```diff
@@ -230,41 +231,50 @@ alongside other information to be represented in a resource object, these keys
 
 #### Relationships <a href="#document-structure-resource-objects-relationships" id="document-structure-resource-objects-relationships" class="headerlink"></a>
 
-The value of the `relationships` key is an object (a "relationships
+The value of the `relationships` key **MUST** be an object (a "relationships
 object"). Members of the relationships object ("relationships") represent
-references from the resource in which it's defined to other resources.
+references from the resource object in which it's defined to other resource
+objects.
 
 Relationships may be to-one or to-many.
 
-The value of a relationship **MUST** be an object (a "relationship object"),
-which **MUST** contain at least one of the following:
+A "relationship object" **MUST** contain at least one of the following:
 
 * `links`: a [links object][links] containing at least one of the following:
   * `self`: a link for the relationship itself (a "relationship link"). This
     link allows the client to directly manipulate the relationship. For example,
     it would allow a client to remove an `author` from an `article` without
     deleting the `people` resource itself.
-  * `related`: a "related resource link", as defined below.
-* `data`: representing "resource linkage", as defined below.
-* `meta`: a [meta] object that contains non-standard meta-information about the
+  * `related`: a [related resource link]
+* `data`: [resource linkage]
+* `meta`: a [meta object][meta] that contains non-standard meta-information about the
   relationship.
 
 A relationship object that represents a to-many relationship **MAY** also contain
 [pagination] links under the `links` member, as described below.
 
-A "related resource link" provides access to the resources targeted by the
-relationship. When fetched, it returns the related resource object(s) as the
-response's primary data. For example, an `article`'s `comments` relationship
-could specify a link that returns a list of comment resource objects when
+#### Related Resource Links <a href="#document-structure-resource-objects-related-resource-links" id="document-structure-resource-objects-related-resource-links" class="headerlink"></a>
+
+A "related resource link" provides access to [resource objects] [linked][links]
+in a [relationship]. When fetched, the related resource object(s) are returned
+as the response's primary data.
+
+For example, an `article`'s `comments` [relationship] could specify a
+[link][links] that returns a collection of comment [resource objects] when
 retrieved through a `GET` request.
 
-A related resource link **MUST** remain constant even when the relationship
+A related resource link **SHOULD** remain constant even when the relationship
 (the set of referenced resources) mutates. That is, the response from a
```

Comment:
I know you're trying to guard against redirects and such. I think the following gets to the heart of the requirement: 

> A related resource link **MUST NOT** change because the associated relationship's membership changes.